src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -931,7 +931,7 @@ class LedgerMasterImp
             mAdvanceWork = false; // If there's work to do, we'll make progress
             bool progress = false;
 
-            std::list<Ledger::pointer> pubLedgers = findNewLedgersToPublish ();
+            auto pubLedgers = findNewLedgersToPublish ();
```

Comment:
I don't see `pubLedgers` being mutated.  My personal inclination is to declare anything that I'm not mutating right now `const`.  There are those who disagree with that approach...  I'll just put it out there for your consideration.